MathGeoLib.Managed.App/Program.cs

```diff
@@ -26,12 +30,22 @@ float Get()
             var numFaces = OrientedBoundingBox.NumFaces;
             var numVertices = OrientedBoundingBox.NumVertices;
 
+#if UNITY_EDITOR || UNITY
+            var box = new OrientedBoundingBox(Vector3.zero, Vector3.zero, Vector3.right, Vector3.up, Vector3.forward);
+#else
```

Comment:
@PiSoup
Just one thing, since you've made this conditional, I suppose then you might want to import `System.Numerics` when it isn't for Unity.

Don't feel obliged to update the sample, it's ok! 😃

Reply:
If I end up adding more to the fork I'll do that. It compiled and ran in both Unity and VS2017 but I'm sure there is no shortage of environments that would break it.